pages/glossary/context.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Context
+key: context
+---
+
+The context of some piece of information is any other information that helps a user understand it.
```

Comment:
The context of some piece of information is any other additional information that helps a user understand the original piece of information.

haha its like a tongue twister :)

Reply:
Is this better or worse? 
The context of a piece of information is any other information that helps a user understand it by clarifying its meaning.